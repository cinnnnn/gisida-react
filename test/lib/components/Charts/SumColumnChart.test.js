import React from 'react';
import { SumColumnChart }  from '../../../../src/lib/components/Charts/SumColumnChart.jsx'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const calcChartWidth = jest.fn();
const moveMapLedgend = jest.fn();

const componentWrapper = shallow(
	<SumColumnChart 
		mapId='map-1'
		layerId=''
		layerData={{}}
		chartSpec={{}}
		layer={{}}
		children={{}}
		locations={{}}
		chartHeight={30}
		chartWidth={30}
		isChartMin={false}
		isFullBleed={false}
		isPrimary={false}
		calcChartWidth={calcChartWidth}
		moveMapLedgend={moveMapLedgend}
	/>
);

describe('SumColumnChart', () => {
	it('SumColumnChart component renders correctly', () => {
		const json = toJson(componentWrapper)
		expect(json).toMatchSnapshot();
	})
});