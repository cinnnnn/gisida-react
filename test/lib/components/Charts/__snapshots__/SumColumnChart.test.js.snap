// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SumColumnChart SumColumnChart component renders correctly 1`] = `
<div
  className="column"
  style={
    Object {
      "bottom": "0px",
      "height": "30px",
      "right": "340px",
      "width": 30,
    }
  }
>
  <ColumnChart
    categories={Array []}
    chartHeight={30}
    chartWidth={30}
    seriesData={Array []}
    targetMark={1}
  />
</div>
`;
